nixos/services/mail/default.nix

```diff
@@ -399,10 +386,6 @@ in
                   "permit_sasl_authenticated"
                   "reject_unknown_helo_hostname"
                 ];
-                smtpd_tls_mandatory_protocols = lib.mkForce ">=TLSv1.2";
-                smtpd_tls_protocols = lib.mkForce ">=TLSv1.2";
-                smtp_tls_mandatory_protocols = lib.mkForce ">=TLSv1.2";
-                smtp_tls_protocols = lib.mkForce ">=TLSv1.2";
```

Comment:
I checked that this change is okay, as it's now incorporated in simple-nixos-mailserver.